docs/UserGuide.adoc

```diff
@@ -216,371 +275,364 @@ clear-task d/12/09/2019
-means that the index must exist within this list!
+=== CCA tracker
+The CCA tracker is a feature unique to `Jarvis`. Entirely customizable based on individual interest,
+`Jarvis` can support milestone tracking, equipment management and so much more! Never has being in multiple
+CCAs been so easy.
 
+==== Adding a CCA: `add-cca`
+To start, we can add some CCAs to the CCA Tracker. You will need to to specify
+the name of the cca and the type of CCA.
 
-==== Edit an installment: `edit-install`
-Spotify just released a new student plan? Great, that means a lower subscription fee every month!
+Format: `add-cca n/NAME t/TYPE [e/EQUIPMENT]...`
 
-Let's inform Jarvis of the changes using the index of the installment on the list __What I spend on monthly...__ under
-the Finance tab. You can either change the description or the amount or both.
+The `types` of the CCA are:
 
-Format: `edit-install INDEX {d/INSTALLMENT-DESCRIPTION | a/AMOUNT}`
+* `performingArt`
+* `sport`
+* `club`
+* `uniformedGroup`
 
 ===== Example
 ----
-edit-install 1 a/10
-edit-install 2 d/Spotify student plan
-edit-install 2 d/Spotify student plan a/7.50
+add-cca n/Swimming t/sport e/swimming trunks e/goggles
+add-cca n/Canoeing t/sport
+add-cca n/Guitar Ensemble t/performingArt e/guitar
 ----
 
+==== Deleting a CCA: `delete-cca`
+New semester, new CCAs - deleting outdated ones is a breeze.
 
-==== Setting a monthly spending limit: `set-limit`
-It's the season to stop spending so much!
+Format: `delete-cca INDEX`
 
-Let Jarvis keep track of how much you have limited yourself to spend this month. Simply set the limit to the
-Finance Tracker.
-
-Format: `set-limit a/AMOUNT`
+where `INDEX` is the *one-based* of a CCA.
 
 ===== Example
 ----
-set-limit a/300
-set-limit a/500.90
+delete-cca 1
 ----
 
+==== Editing a CCA: `edit-cca`
+If you have bought some new running shoes, or the latest tuner for your violin, let `Jarvis`
+know so that your changes to your CCAs can be updated!
 
-==== Listing finances: `list-finances`
-Jarvis will switch you onto the Finance tab if you are not already there. If you have previously searched through your
-purchases for something, then Jarvis returns you to the original list of purchases.
-
-Format: `list-finances`
-
-
-==== Export to File: `export` - coming to you in v2.0!
-Want to keep a record of your finances for this month?
-
-Simple export your current expenditures to a file, either
-as a `.txt`, `.csv` or `xlsx` format.
-
-Format: `export {csv | txt | xlsx} f/FILENAME`
+Format: `edit-cca INDEX { n/NAME | t/TYPE | e/EQUIPMENT...}`
 
 ===== Example
 ----
-export csv f/expenditures
+edit-cca 1 t/sport e/tennis racket
+edit-cca 1 t/performingArt
+edit-cca 1 n/Running e/shoes e/water bottle
 ----
 
-NOTE: Jarvis will create a file called `expenditures.csv` in the current directory, containing all expenditures.
+[NOTE]
+`Jarvis` will soon also be able to edit the milestones of your CCAs in v2.0!
```

Comment:
Can we standardise how the features are labelled v2.0? Currently they differ across planner, cca tracker and finance tracker.